engine/spverrors/definitions.go

```diff
@@ -375,3 +375,18 @@ var ErrBroadcastWrongBUMPResponse = models.SPVError{Message: "broadcast server r
 
 // ErrBroadcastRejectedTransaction is when broadcast server rejected transaction
 var ErrBroadcastRejectedTransaction = models.SPVError{Message: "broadcast rejected transaction", StatusCode: 400, Code: "error-broadcast-rejected-transaction"}
+
+// ErrARCUnreachable is when ARC cannot be requested
+var ErrARCUnreachable = models.SPVError{Message: "ARC cannot be requested", StatusCode: 404, Code: "error-broadcast-unreachable"}
```

Comment:
maybe code 500 here? I'm not sure about this but 404 might indicate that we tried an endpoint that doesn't exist. WDYT?

Reply:
The status code is not-applicable for this error.
This is why we've recently been discussing how to refactor it.
But sure 👍 
500 would look better - but, whatever 😄